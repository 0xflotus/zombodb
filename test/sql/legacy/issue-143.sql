SELECT zdb. count('idxso_posts', '({"term": {"body": "java"}})');