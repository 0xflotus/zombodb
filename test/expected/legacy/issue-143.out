SELECT zdb. count('idxso_posts', '({"term": {"body": "java"}})');
 count 
-------
 13310
(1 row)

